overseerr-api.yml

```diff
@@ -6939,3 +6947,4 @@ paths:
 security:
   - cookieAuth: []
   - apiKey: []
+  - forwardAuth: []
```

Comment:
This is not in the OpenAPI spec?
https://spec.openapis.org/oas/latest.html#security-scheme-object-0